app/src/ui/history/compare.tsx

```diff
@@ -117,12 +117,17 @@ export class CompareSidebar extends React.Component<
 
   public render() {
     const formState = this.props.compareState.formState
+    const numBranches = this.props.compareState.allBranches.length
     const placeholderText =
-      formState.kind === ComparisonView.None
+      numBranches <= 1
         ? __DARWIN__
-          ? 'Select Branch to Compare...'
-          : 'Select branch to compare...'
-        : undefined
+          ? 'There Are No Branches to Compare'
```

Comment:
Sorry about the nested ternary. If there were a function that could handle casing for me, I could remove the nesting.

Reply:
I think `Are` should be lower-cased here. But I'm also :thumbsup: to `No Branches to Compare` here which would make that change unnecessary.

Reply:
Most style guides ask for all verbs to be capitalised in titles and headings. _"Capitalize My Title"_ is a great tool when in doubt - https://capitalizemytitle.com

Reply:
@japan thanks for the tip - that will definitely help for the future!